FrameworkSystem/private/standardLogging/Logging.py

```diff
@@ -36,7 +36,7 @@ class Logging(object):
   # it can be composed by the system name and the component name. For instance: "Monitoring/Atom"
   _componentName = "Framework"
   # lock the configuration of the Logging
-  _lockConfig = threading.RLock()
+  _lockConfig = LockRing().getLock("config")
```

Comment:
Instead of calling the init everytime, just call once LockRing and store it as an attribute.